src/main/java/racingcar/domain/car/Car.java

```diff
@@ -0,0 +1,26 @@
+package racingcar.domain.car;
+
+public class Car {
+
+    private final String name;
+
+    private int distance = 0;
+
+    public Car(String name) {
+        this.name = name;
+    }
```

Comment:
정적 팩토리 메서드를 사용해보시는건 어떠신가요!?
```suggestion
    private Car(String name) {
        this.name = name;
    }
    
    public static Car create(String name) {
        return new Car(name);
    }
```

정적 팩토리 메서드는 객체 생성 로직을 클래스 내부에 숨기는 역할 즉, 캡슐화를 통해 코드를 객체 지향적으로 작성하는 데 도움을 주기 때문에 적용해 보는 것도 좋을 것 같아요!

Reply:
이 부분은 저도 생각하지 못했던 부분이네요! 저도 배워갑니다! ㅎㅎ

Reply:
헉... 정말 생각하지 못했던 부분이네요
저렇게 생성자 자체를 `private`으로 숨길 수 있군요..!
좋은 사실을 알고 갑니다!
감사합니다!! 🥰